pyproject.toml

```diff
@@ -11,6 +11,7 @@ dependencies = [
 
 [tool.commitizen]
 bump_message = "bump: v$current_version → v$new_version"
+hooks_pre_bump = ["uv lock --offline"]
```

Comment:
Open question: is `uv.lock` automatically staged? To be tested. If not, we need to do `&& git add uv.lock` as well

Reply:
Good question, I don't know. To be tested!

Reply:
Tested and it doesn't work. I'm trying out some solutions, keep you posted 🙂